example-config.yml

```diff
@@ -338,12 +338,17 @@ itinerary:
 ### The nested structure should be the same as the language files under the i18n folder.
 ### You can also customize OTP error messages for itinerary searches based on OTP HTTP codes.
 ### A separate message can be set for each language or locale if necessary.
+### The LocaleSelector component, which provides a dropdown for the user to change their locale, will
+### only be rendered if more than one valid language (all languages must have "name" defined) is included
+### in the language config. Locale icons (flags) are currently defined in locale-selector.tsx file. 
+
 # language:
 #   allLanguages
 #     common:
 #       accessModes:
 #         bikeshare: Relay Bike
 #   en-US:
+#     name: English (US)
```

Comment:
I think I get the need to list supported languages, so this could be a way to do so. I am puzzled about providing the display names of each language. There is functionality available with react-intl already: https://formatjs.io/docs/react-intl/api#formatdisplayname (we may have to use the polyfill also). Let me think through it.

Reply:
The polyfill looks like it's massive! As cool as that feature would be, maybe it's a good idea to wait until intl has broad browser support...

Reply:
Okay. We don't need to have that in this PR.